fission/src/mirabuf/MirabufSceneObject.ts

```diff
@@ -295,6 +313,22 @@ class MirabufSceneObject extends SceneObject implements ContextSupplier {
 
         this.updateBatches()
 
+        // Only for game piece imported with a field
+        if (this._mirabufInstance.parser.isGamePiece) {
+            const jBodyId = this.mechanism.getBodyByNodeId(this.mechanism.rootBody)
+            if (!jBodyId) {
+                console.warn(
+                    `Jolt Body for SceneObject ${this.id} with rootBody ${this.mechanism.rootBody} as NodeId not found`
+                )
+                return
+            }
+            const position = convertThreeVector3ToJoltRVec3(
+                new THREE.Vector3().setFromMatrixPosition(this.mirabufInstance.parser.gamePieceTransform!)
+            )
+            World.physicsSystem.setBodyPosition(jBodyId, position)
+            this.updateMeshTransforms()
+        }
```

Comment:
It looks like this is happening with independently imported gamepieces too

<img width="1156" height="660" alt="Image" src="https://github.com/user-attachments/assets/6a471dfa-8cbb-4ee4-9f3a-f81a37bd4231" />

Reply:
yeah they're spawning where they would go if they were part of the field rather than at the center